pandas/tests/test_panel.py

```diff
@@ -2,11 +2,14 @@
 # pylint: disable=W0612,E1101
 
 from datetime import datetime
-from inspect import getargspec
+try:
+  from inspect import signature
```

Comment:
I have tried to do that in  Update __init__.py #12117 .
Do I need to make any other changes?